rero_ils/modules/entities/local_entities/subclasses/place.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""API for manipulating "place" local entities."""
+
+from ..api import LocalEntity
+
+
+class PlaceLocalEntity(LocalEntity):
+    """Person local entity class."""
+
+    def get_authorized_access_point(self):
+        """Get the authorized access point this local entity.
+
+        :return return the calculated authorized access point to use.
+        """
+        return self.get('name')
```

Comment:
Idea: Maybe this generic definition could be the basic `LocalEntity. get_authorized_access_point` implementation (then remove ABC). It will save 3 classes at this time (place, temporal, topic).

PS : Yes I know.... I'm schizophrenic